charts/super-agent-deployment/values.yaml

```diff
@@ -97,18 +97,67 @@ fedramp:
 verboseLog:
 
 config:
-  # -- Set if the configMap is going to be created by this chart or the user will provide its own.
-  create: true
-  # -- The name the configMap is going to have. If create is set to false, the name of an existing configMap that will be used to config the Super Agent.
-  name: ""
-  # -- The key in the configMap that has the configuration for the Super Agent.
-  key: ""
-  # -- Here you can set New Relic' Super Agent configuration.
+  # -- Configuration for the Super Agent.
+  # @default -- See `values.yaml`
+  superAgent:
+    # -- Set if the configMap is going to be created by this chart or the user will provide its own.
+    create: true
+    # -- The name the configMap is going to have. If create is set to false, the name of an existing configMap that will be used to config the Super Agent.
+    name: ""
+    # -- The key in the configMap that has the configuration for the Super Agent.
+    key: ""
+    # -- Here you can set New Relic' Super Agent configuration.
+    # @default -- See `values.yaml` for examples
+    content:
```

Comment:
to help clarify would help to change:
 - content -> configFile
 - subAgents.{agent_name} -> subAgents.{agent_name}.variableFile 
 ?
 
 So that it would look like:
```
config:
  superAgent:
     ...
    configFile: {}
  subAgent:
    name_agent1:
       variableFile: {}
    name_agent2:
       variableFile: {}
```

Reply:
I am not sure, just thinking how to show how the config looks like

Reply:
fine for me

Reply:
Done on [#2b89922](https://github.com/newrelic/helm-charts/pull/1221/commits/2b899228f79b208679b3e75a437d7132d642b4e5)